.github/workflows/test.yml

```diff
@@ -4,16 +4,19 @@ on:
   push:
     branches: [ master ]
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - milestoned
     branches: [ master ]
-  pull_request_review:
-      types: [ submitted ]
 
 jobs:
   build:
     # Trigger a run when approving a bot PR or as usual
     if: >-
-      (github.event.review.state == 'approved' && contains(fromJSON('["dependabot[bot]", "github-actions[bot]"]'), github.event.pull_request.user.login)) ||
-      (github.actor != 'dependabot[bot]' && github.event_name != 'pull_request_review')
+      (github.event.review.state == 'approved' && contains(fromJSON('["atchertchian", "github-actions[bot]"]'), github.event.pull_request.user.login)) ||
+      (github.actor != 'atchertchian' && github.event_name != 'pull_request_review')
```

Comment:
I did this blindly just to test (this is a temporary change, see [tmp] commit) but I'm not actually sure we should skip on dependabot PRs: maybe we should actually fail (using the check-actions-source new action?)

Similarly maybe we want to align the dependabot process on this repo and trigger on milestone (and maybe setup auto-merge on approval (?))

Thoughts welcome @gionn @dsibilio

Reply:
will test on another repo (will revert temp changes)